msg/SpawnEntityState.msg

```diff
@@ -0,0 +1,4 @@
+# Holds an entity state, xml description and UE tags
+string xml                  # Entity XML description as a string : entity type, etc
```

Comment:
Do we need xml?

Reply:
I suppose we could rename it to `ModelName` for its being used as [entity key name](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/Source/RapyutaSimulationPlugins/Private/Tools/SimulationState.cpp#L256) which essentially represents an object/robot's ModelName.
Also, since our Sim now only requires users to only provide a path to their own master folder of URDF/SDF models, &  automatically load their contents under the hood, thus they would not have to pass the bulky XML content in each service call's message.

Reply:
I added a new message so that the service uses an array of this new type, especially as the message has an array for tags.  

The xml variable in SpawnEntityState.msg is like the one used in SpawnEntity service. Actually it's the type of object to spawn without any xml formatting so I could either change the variable name, or change its processing in the service in order to use xml formatting.  

The SpawnEntities service also has an xml variable which is not used for now but I added it in case it could be useful afterwards

Reply:
`SpawnEntities`'s xml is quite a legacy name, when we intended to pass the whole xml content through the message I guess. So it should be also renamed for clarity in the handling on Sim side, since our Sim should do the xml loading itself as already did, allowing users to only need to pass the Entity Model name only, which is also more economic.

Reply:
I decided to simplify the service in order to remove the new msg.
There are some bug/limitations in the code generator, for example, the `tags` string array was "translated" as a simple string variable.
In this case, the need for a new message is less obvious.

So now, each entity is defined by a type, state and tag.